presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -571,27 +572,12 @@ public static long stNumGeometries(@SqlType(GEOMETRY_TYPE_NAME) Slice input)
         return ((OGCGeometryCollection) geometry).numGeometries();
     }
 
-    @Description("Returns a geometry that represents the point set union of the input geometries. This function doesn't support geometry collections.")
+    @Description("Returns a geometry that represents the point set union of the input geometries.")
```

Comment:
Docs need to be updated to remove `This function doesn’t support geometry collections.` from `ST_Union` as well.

Reply:
I did that, but it got squished into the other commit.  I'll fix it so it's in the right commit.